doc/content/guides/getting-started/console/_index.md

```diff
@@ -0,0 +1,119 @@
+---
+title: "Console"
+description: ""
+weight: 8
+---
+
+<style>
+img {
+  border: 1px solid #ddd;
+}
+</style>
+
+## Login
+
+You need to be logged in inside the Console in order to create gateways, applications, devices and API keys. With the stack running in one terminal session, open https://localhost:8885 in your browser.
+
+>Note: If you are hosting your stack on a remote machine, you should replace `https://localhost:8885` with your own host address and port.
+
+{{< figure src="front.png" alt="Front page" >}}
+
+You are now on the Console landing page. Click `Login with your TTN Stack Account` in order to reach the login page.
+
+{{< figure src="login.png" alt="Login" >}}
+
+After entering your credentials and logging in, you will reach the Console overview page. You can now fully exploit the capabilities of the Console.
+
+{{< figure src="overview.png" alt="Overview" >}}
+
+## Create gateway
+
+Click on `Register a gateway` in the overview page in order to reach the gateway registration page. Fill the gateway ID, gateway EUI, gateway name and frequency plan and click `Create Gateway`.
+
+{{< figure src="gateway-creation.png" alt="Gateway creation" >}}
+
+Your gateway will be created and you will be redirected to the gateway overview page of your newly created gateway. 
+
+{{< figure src="gateway-overview.png" alt="Gateway overview" >}}
+
+You can now connect your gateway to the stack.
```

Comment:
We don't have a gateway connection guide yet, right?

Reply:
Don't quite get the sentence here. What is meant by connecting the gateway to the stack? Technically the gateway is connected to the stack after registration, isn't it?

Reply:
Strictly speaking it is "configuring the gateway to connect to the stack". This is copied from the CLI guide, so let's keep it the same.